src/ensembl/src/styles/_settings.scss

```diff
@@ -48,6 +49,7 @@ $global-box-shadow: $medium-light-grey;
   ice-blue: $ice-blue;
   red: $red;
   orange: $orange;
+  mustard: $mustard;
```

Comment:
I suppose if this color deserves its own name, we should add it to Storybook then?

https://github.com/Ensembl/ensembl-client/blob/dev/src/ensembl/stories/design-primitives/colours/Colours.stories.tsx